### DISCUSSION TITLE
Setting taskwiki markup syntax

### DISCUSSION THREAD

**Discussion by muraii** [tbabej/taskwiki#342] [Q&A] [answered]
I hope this isn't splintering feedback into too many different places, but my question isn't really an _issue_. In [issue #206](https://github.com/tools-life/taskwiki/issues/206), the setting `g:taskwiki_markup_syntax = 'markdown'` is mentioned. I can't find this referenced in any documentation, nor a discussion of different markup syntaxes generally. Is this still a thing?

**Comment by tbabej** [comment, marked as answer]
It is still a thing, actually, it is the markup syntax I am personally currently using :slightly_smiling_face: 

```vim
nnoremap <silent><buffer> <LocalLeader>t :TaskWikiMod type:
let g:taskwiki_markup_syntax = "markdown"
```
